artifactory/commands/oc/startbuild.go

```diff
@@ -158,19 +161,44 @@ func (osb *OcStartBuildCommand) validateOcVersion() error {
 }
 
 func startBuild(executablePath string, ocFlags []string) (ocBuildName string, err error) {
-	cmdArgs := []string{"start-build", "-w", "--template={{.metadata.name}}"}
+	cmdArgs := []string{"start-build", "-w", "--template={{.metadata.name}}{{\"\\n\"}}"}
 	cmdArgs = append(cmdArgs, ocFlags...)
 
-	outputBytes, err := exec.Command(executablePath, cmdArgs...).Output()
+	log.Debug("Running command: oc", strings.Join(cmdArgs, " "))
+	cmd := exec.Command(executablePath, cmdArgs...)
+	outputReader, err := cmd.StdoutPipe()
+	if err != nil {
+		return "", errorutils.CheckError(err)
+	}
+	defer func() {
+		e := errorutils.CheckError(outputReader.Close())
+		if err == nil {
+			err = e
+		}
+	}()
+	cmd.Stderr = os.Stderr
+	err = cmd.Start()
+	if err != nil {
+		return "", errorutils.CheckError(err)
+	}
+
+	// The build name is in the first line of the output
+	scanner := bufio.NewScanner(outputReader)
+	scanner.Scan()
+	ocBuildName = scanner.Text()
+
+	// Print the output to stderr
+	_, err = io.Copy(os.Stderr, outputReader)
```

Comment:
I think that the above code can lead to a situation where the stderr and stdout lines aren't prompted at the right order. This is becuase the following line - 
```
_, err = io.Copy(os.Stderr, outputReader)
```
prints the entire ```outputReader``` separately from the stderr lines.

Reply:
It will be fixed in another PR. [I opened an issue for it.](https://github.com/jfrog/jfrog-cli-core/issues/254)